oktapam/resource_database.go

```diff
@@ -0,0 +1,380 @@
+package oktapam
```

Comment:
should this file be in the PR?  it seems like its real definition is within #122

Reply:
Sorry, I pinged Sachin about this PR but not you.  This is rebased on 122 since it uses some of the same changes.  I figured it could be merged first then this one should be easy and hopefully not need another rebase.